Package.swift

```diff
@@ -129,19 +129,46 @@ func isDebuggingMain(_ isDebug: Bool) {
     }
 }
 
+func xcodeFolder() -> String {
+    (Executable("/usr/bin/xcode-select")("-p") ?? "").trimmed
```

Comment:
This could theoretically run on a computer without Xcode, right? Like what if someone just installs the command line tools? Or is that a concern for a future where we can actually start using the new SPM-based SwiftSyntax?

Reply:
yes, it can run without Xcode installed, as far as I know

Reply:
Will that be a problem for this line?

Reply:
maybe?
but this code is there for a long time and I don't remember seeing any issues...so it's probably safe

we can validate that (later) by deleting Xcode and just installing the swift compiler